module3/index.html

```diff
@@ -0,0 +1,143 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+        <title>STL - złożoność obliczeniowa, iteratory i algorytmy - Coders School</title>
+
+        <meta name="description" content="STL">
+        <meta name="author" content="Mateusz Adamski & Łukasz Ziobroń">
+
+        <link rel="stylesheet" href="../../css/reset.css">
+        <link rel="stylesheet" href="../../css/reveal.css">
+        <link rel="stylesheet" href="../../css/theme/coders.css" id="theme">
+
+        <!-- Theme used for syntax highlighting of code -->
+        <link rel="stylesheet" href="../../lib/css/monokai.css">
+
+        <!-- Printing and PDF exports -->
+        <script>
+            var link = document.createElement( 'link' );
+            link.rel = 'stylesheet';
+            link.type = 'text/css';
+            link.href = window.location.search.match( /print-pdf/gi ) ? '../../css/print/pdf.css' : '../../css/print/paper.css';
+            document.getElementsByTagName( 'head' )[0].appendChild( link );
+        </script>
+    </head>
+    <body>
+        <div class="reveal">
+            <div class="slides">
+                <section>
+                    <section data-background="#111111">
+
+                        <h1>STL #3</h1>
+                        <h2>Złożoność obliczeniowa, iteratory, algorytmy</h2>
+                        <a href="https://coders.school">
+                            <img width="500" data-src="../coders_school_logo.png" alt="Coders School" class="plain">
+                        </a>
+                        <h3>Mateusz Adamski</h3>
+                        <h3>Łukasz Ziobroń</h3>
+
+                    </section>
+                    <section data-markdown>
+                        <textarea data-template>
+
+                        ## Zadania
+
+                        Repo GH `coders-school/stl`
+
+                        [https://github.com/coders-school/stl/tree/master/module3](https://github.com/coders-school/stl/tree/master/module3/homework)
```

Comment:
```suggestion
                        [Repo GH `coders-school/stl`](https://github.com/coders-school/stl/tree/master/module3/homework)
```

Zrobiłbym to w taki sposób, ale problem jest natury innej. Link nie działa. 404. Trzeba by było zaktualizować link.